### ISSUE TITLE
conflict/bug?:  object_info only contains one "Mask Combine" when both WAS and Masquerade installed

### ISSUE THREAD

**Issue by rvion** [comfyanonymous/ComfyUI#630]
🔴 
both [was-node-suite-comfyui](https://github.com/WASasquatch/was-node-suite-comfyui) and [masquerade-nodes-comfyui](https://github.com/BadCafeCode/masquerade-nodes-comfyui) export 

- https://github.com/BadCafeCode/masquerade-nodes-comfyui/blob/d7af5106b3e5347d20202d7768bb311d2afd7203/MaskNodes.py#L1022
<img width="585" alt="image" src="https://user-images.githubusercontent.com/2150990/236660617-0b12c217-a940-49c1-b6a1-9978fd8e6ae9.png">

- https://github.com/WASasquatch/was-node-suite-comfyui/blob/main/WAS_Node_Suite.py#L8342
<img width="628" alt="image" src="https://user-images.githubusercontent.com/2150990/236660612-21919b5a-4667-4782-bde1-224e723cf341.png">

 `/object_info` only contains one "Mask Combine" when both WAS and Masquerade installed

![image](https://user-images.githubusercontent.com/2150990/236660466-43f37fd8-c8e3-44ef-9134-84cc516166e8.png)

I really think this is a bug; since custom_node authors may not be aware of each other

🔴 this situation is bound to happen more and more often as time pass, because lots of custom_node suites will re-use the same sames naturally as ComfyUI becomes popular. 

A schema change is probably needed sooner than later. a way to disambiguate both

**Comment by rvion** [comment]
![image](https://user-images.githubusercontent.com/2150990/236660995-742fedc8-1680-4f22-8647-bb5ab3cd6143.png)
?

**Comment by rvion** [comment]
I didn't explicity state it in the issue, but the problem is that workflow can suddently stop working, as some custom_node update can silently override some other nodes. 

I think this may require a breaking change, but I also think that if a breaking change is really required, it really is worth it in this case

**Comment by WASasquatch** [comment]
Looks like that's just my bad. I didn't consider the simplicity of the name VS it's likely use. 

This is specifically why I suggested unique names should occur from the class name since we often uniquely name the classes so they don't collide program wise. Like mine would be `WAS_Combine_Masks`

**Comment by comfyanonymous** [comment]
Yeah this was fine for the internal nodes but for custom nodes we are going to have to force custom nodes to have something like a unique prefix.

**Comment by WASasquatch** [comment]
> Yeah this was fine for the internal nodes but for custom nodes we are going to have to force custom nodes to have something like a unique prefix.

What's so bad about just using the class names? It's already standard practice to uniquely name your packages/modules, especially when used as extensions to another program for obvious reasons, and so far everyone is doing that. So there is your unique ID.

**Comment by rvion** [comment]
1. 👉I would prefer having an **authority** part, like `author` rather than UUID or manual people coordination.
2. 👉I prefer all similar nodes to have similar names. better to have multiple `loadImage` nodes than a mix of `ImageLoad`, `LoadImage` and `ImageOpen` due to conflict consideration.
3. In CushyStudio, I'll probably generate classNames with suffix rather than prefix to disambiguate things so it play well with autocomplete
      4. LoadImage
      5. LoadImage_WAS
      6. LoadImage_Rvion
4. manual coordination to avoid disambiguation will lead to antipatterns.
5. bonus remark: looking elsewhere, most thriving communities have a a scope/authority/author hierarchy system to allow everyone to share names: github, docker hub, etc. allowing the same names also promotes node forking, like on github

```diff
{ 
-    "Combine Masks": {
+    "WAS/Combine Masks": {
        "input": {... },
        "output": ["MASK"],
        "output_name": ["MASK"],
        "display_name": "Combine Masks",
+       "author": "WAS", 🟢
        "name": "Combine Masks",
        "description": "",
        "category": "WAS Suite/Image/Masking"
    },
}
```

**Comment by WASasquatch** [comment]
> 1. 👉I would prefer having an **authority** part, like `author` rather than UUID or manual people coordination.
> 
>     2. 👉I prefer all similar nodes to have similar names. better to have multiple `loadImage` nodes than a mix of `ImageLoad`, `LoadImage` and `ImageOpen` due to conflict consideration.
> 
>     3. In CushyStudio, I'll probably generate classNames with suffix rather than prefix to disambiguate things so it play well with autocomplete
>        4. LoadImage
>        5. LoadImage_WAS
>        6. LoadImage_Rvion
> 
>     4. manual coordination to avoid disambiguation will lead to antipatterns.
> 
>     5. bonus remark: looking elsewhere, most thriving communities have a a scope/authority/author hierarchy system to allow everyone to share names: github, docker hub, etc. allowing the same names also promotes node forking, like on github
> 
> 
> here is a proposed suggestion related to the `object_info` format :
> 
> ```diff
> { 
> -    "Combine Masks": {
> +    "WAS/Combine Masks": {
>         "input": {... },
>         "output": ["MASK"],
>         "output_name": ["MASK"],
>         "display_name": "Combine Masks",
> +       "author": "WAS",
>         "name": "Combine Masks",
>         "description": "",
>         "category": "WAS Suite/Image/Masking"
>     },
> }
> ```

The immediate issue I see with this is some people like me, don't use the node names system, as it's extra work for 130+ nodes that's ridiculous to have to do imo. 

So wouldn't that means all my nodes would have "WAS/Combine Masks" type names, when I already put them in their own menu to designate them? Or is this purely CushyStudio?

**Comment by rvion** [comment]
- I think I would rather be in favor of removing the need for this extra mapping object, and using class Names by default
- I'm only requesting adding a single variable `author = "WAS"` somewhere, and nothing more, so object_info can include this.

node: the `author` system and key prefix in object_info is **backward compatible** if optional 
- no author => same key as now,
- author => prefixed key in `/object_info` + author key
